_assets/borg.css

```diff
@@ -21,6 +21,45 @@ header h1 {
 	text-align: center;
 }
 
+nav {
+	display: flex;
+	justify-content: center;
+}
+
+nav > ol {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 1rem;
+	padding: 0;
+}
+
+nav > ol > li {
+	list-style: none;
+}
+
+/* mono system font stack courtesy of https://qwtel.com/posts/software/the-monospaced-system-ui-css-font-stack/ */
+nav > ol > li > a {
+	font-size: 1.1rem;
+	font-weight: bold;
+	color: #22D045;
+	font-family: ui-monospace, 
+		Menlo, Monaco, 
+		"Cascadia Mono", "Segoe UI Mono", 
+		"Roboto Mono", 
+		"Oxygen Mono", 
+		"Ubuntu Monospace", 
+		"Source Code Pro",
+		"Fira Mono", 
+		"Droid Sans Mono", 
+		"Courier New", monospace;
```

Comment:
Guess courier new might be a bit ugly as it is with serifs, but considering it is only a fallback if nothing else is found I guess it is acceptable.

Reply:
"Courier New" seems to be quite different from "Courier", looks like a sans-serif to me:

![Screenshot from 2023-08-24 12-46-02](https://github.com/borgbackup/borgbackup.github.io/assets/13798767/89966c7c-666f-4f9a-9a5b-558d5b34b6bd)

Reply:
Do you have "courier new" installed?

https://learn.microsoft.com/en-us/typography/font-list/courier-new there it looked differently.

Reply:
done